package.json

```diff
@@ -80,6 +80,7 @@
   },
   "dependencies": {
     "opn": "5.3.0",
-    "react": "^16.13.1"
+    "react": "^16.13.1",
+    "source-map": "^0.7.3"
```

Comment:
Oh whoops, neither of these should be dependencies, they should only be `devDependencies` because it's needed to build speedscope but isn't needed to run speedscope as an npm module (the npm module only exposes a binary, not a JavaScript API).

I'm actually not sure why `react` is in here at all 😬 , but that's a mistake.

So as part of this PR, can you remove `react`, move `source-map` to `devDependencies`, and also commit the changes to `package-lock.json`?